EVGEN/AliGenReaderHepMC.cxx

```diff
@@ -59,7 +60,8 @@ Int_t AliGenReaderHepMC::NextEvent()
       fParticleIterator->Reset();
       THepMCParser::HeavyIonHeader_t heavyIonHeader;
       THepMCParser::PdfHeader_t pdfHeader;
-      THepMCParser::ParseGenEvent2HeaderStructs(fGenEvent,heavyIonHeader,pdfHeader,true,true);
+      THepMCParser::CrossSectionHeader_t csHeader;
+      THepMCParser::ParseGenEvent2HeaderStructs(fGenEvent,heavyIonHeader,pdfHeader,csHeader,true,true,true);
```

Comment:
Maybe it is an effect of the way the change is visualized, but can you check the indentation?

Reply:
Same comment concerning the indentation as above.